js/main.js

```diff
@@ -86,36 +92,47 @@ function getArray(array) {
     }
   }
   return arrayRandom;
-}
-
-const adsNearby = () => {
-  const getRandomArrayElement = (elements) => elements[getRandomPositiveInteger(0, elements.length - 1)];
-  let randomPhotoIndex = getRandomPositiveInteger(1, 10);
-  const randomPrice = getRandomPositiveInteger (0, 10000000);
-  const randomRooms = getRandomPositiveInteger (0, 1000);
-  const randomGuests = getRandomPositiveInteger (0, 1000);
-  const randomLat = getRandomPositiveFloat (35.65000, 35.70000);
-  const randomLng = getRandomPositiveFloat (139.70000, 139.80000);
-
-  if (randomPhotoIndex < 10) {
-    randomPhotoIndex = `0${ randomPhotoIndex}`;
-  }
+};
+
+const getRandomArrayElement = (elements) => elements[getRandomPositiveInteger(0, elements.length - 1)];
+
+const makeAvatarIndex = () => {
+  let count = 0;
+
+  return function() {
+    count++;
+    if (count < 10) {
+      count = `0${count}`;
+    }
+
+    return count;
+  };
+};
+
+const counter = makeAvatarIndex();
+
+
+const getAdsNearby = () => {
+  const randomLat = getRandomPositiveFloat (LAT_MIN, LAT_MAX);
+  const randomLng = getRandomPositiveFloat (LNG_MIN, LNG_MAX);
+
 
   return {
-    author: `img/avatars/user${ randomPhotoIndex }.png`,
+    author: {
+      avatar: `img/avatars/user${counter()}.png`,
```

Comment:
Для форматирования можно использовать метод строк padStart (смотреть на mdn)
```suggestion
      avatar: `img/avatars/user${counter().toString().padStart(2, '0')}.png`,
```